pcsx2/Input/WindowsHIDUtility.cpp

```diff
@@ -0,0 +1,82 @@
+/*  PCSX2 - PS2 Emulator for PCs
+ *  Copyright (C) 2002-2023  PCSX2 Dev Team
+ *
+ *  PCSX2 is free software: you can redistribute it and/or modify it under the terms
+ *  of the GNU Lesser General Public License as published by the Free Software Found-
+ *  ation, either version 3 of the License, or (at your option) any later version.
+ *
+ *  PCSX2 is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ *  without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
+ *  PURPOSE.  See the GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License along with PCSX2.
+ *  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "Input/WindowsHIDUtility.h"
+
+namespace WindowsHIDUtility
+{
+	std::vector<HidDeviceInfo> FindHids(u16 vid, u16 pid)
```

Comment:
This code isn't used anywhere else, and is probably unlikely to be, so it's best to make it local to the DS3 input source, rather than adding additional files.